src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useCallback, useMemo, useState } from 'react';
+import debounce from 'lodash.debounce';
```

Comment:
Fix it
<img width="1007" alt="image" src="https://github.com/mate-academy/react_autocomplete/assets/64750958/4ea808b4-880e-415a-9790-92c9951b6909">